src/lib.rs

```diff
@@ -195,6 +196,20 @@ struct WalkDirOptions {
     contents_first: bool,
 }
 
+impl fmt::Debug for WalkDirOptions {
+    fn fmt(&self, f: &mut fmt::Formatter) -> ::std::result::Result<(), fmt::Error> {
+        write!(f, "WalkDirOptions {{ follow_links: {:?}, max_open: {:?}, \
```

Comment:
Do we need to manually implement `Debug` for the `sort_by` field?

There's a handy [`debug_struct`](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct) method on `Formatter` that we could use here to simplify this implementation.